src/main/java/uk/ac/ebi/eva/pipeline/configuration/AnnotationLoaderConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package uk.ac.ebi.eva.pipeline.configuration;
+
+import org.opencb.biodata.models.variant.annotation.VariantAnnotation;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.Scope;
+import org.springframework.data.mongodb.core.MongoOperations;
+import uk.ac.ebi.eva.pipeline.Application;
+import uk.ac.ebi.eva.pipeline.io.writers.VepAnnotationMongoWriter;
+import uk.ac.ebi.eva.utils.MongoDBHelper;
+
+@Configuration
+public class AnnotationLoaderConfiguration {
```

Comment:
It would be more clearly linked to step if named `AnnotationLoaderStepConfiguration`. Or is there a better grouping we could do? :thinking: